### ISSUE TITLE
Validate/Harden Page Issues

### ISSUE THREAD

**Issue by GavMason** [mitre/saf-site-frontend#42]
Known bugs and possible enhancements: 

Bugs:
- [X]  There currently is a bug where when scrolling through the table component and routing over to the modal component, Nuxt saves the scroll behavior so that when opening the modal you render already at the bottom of the page. Luckly Nuxt seems to have fixed this issue in the 3.0.0 stable version. (Current plan to check back on this issue after upgrading our Nuxt version)  Related issue: https://github.com/nuxt/nuxt.js/issues/14544

Enhancements
- [ ]  Add the upcoming 'Guidance scraper database' functionality to auto update certain values.

**Comment by wdower** [comment]
Need @Amndeep7 to fix the header wording for Harden/Validate (clarify language, strike reference to Control Assessment table).

Need @ChrisHinchey to make the Details view of a Harden/Validate code repo cleaner. Presently the component just mashes together tables of profile details. Maybe add ability to scroll through them?

Need @GavMason to replace the reference to the SAF group email with a link to the FAQ page (which should have more built out contact info as per #47)

**Comment by Amndeep7** [comment]
All of this text should be ran by Emily/Eugene.

Harden: `These open-source community-based hardening baselines help to securely configure common system components. MITRE is helping to provide stewardship over these hardening baselines, hosted here and at other community vendor sites. If you are interested in new hardening baselines, learn more in our [help center](redirect to faq#new_hardening_content)!`

New hardening content faq: `Our hardening content is created by using tools like [Terraform](link to it), [blah](link to tool), and [blah](link to tool) in order to apply the recommendations or fixes suggested by STIG, CIS Benchmark, and other guideline authors to the intended environment or application.  To learn more about guidelines see [here](#link to plan page or faq on what a stig is or something) or [contact us](#redirect to top of page with contact info).`

Validate: 
```
These open-source community-based InSpec profiles validate the security of common system components. MITRE is helping to provide stewardship over these profiles, hosted here and at other community vendor sites. If you are interested in new profiles, please see our [training material](link to training page or to the docs) to learn how to develop and contribute or [contact us](#contact section on faq) to see how we can help you create them.

All assessment tests under MITRE SAF are associated with NIST SP 800-53 Security Controls.  Learn more about this document in the [help center](link to faq on what is 800-53).
```

800-53 faq: Copy paste the abstract (don't forget to cite) and then link to the site (https://csrc.nist.gov/publications/detail/sp/800-53/rev-5/final).

**Comment by GavMason** [comment]
> Known bugs and possible enhancements:
> 
> Bugs:
> 
> * 

- [ x ] There currently is a bug where when scrolling through the table component and routing over to the modal component, Nuxt saves the scroll behavior so that when opening the modal you render already at the bottom of the page. Luckly Nuxt seems to have fixed this issue in the 3.0.0 stable version. (Current plan to check back on this issue after upgrading our Nuxt version)  Related issue: [NuxtLink retains scroll position nuxt/nuxt#14544](https://github.com/nuxt/nuxt/issues/14544)

> 
> Enhancements
> 
> * Add the upcoming 'Guidance scraper database' functionality to auto update certain values.
> * Update dark mode color palette  (Possibly offical MITRE dark mode color scheme?)

Edit: After upgrading to the latest stable Nuxt version this scrolling cache problem has been addressed and solved.

**Comment by ChrisHinchey** [comment]
After discussion it was determined this issue was completed
